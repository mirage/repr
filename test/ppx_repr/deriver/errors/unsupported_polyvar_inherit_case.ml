type 'a s = [ `S of int | `R ] [@@deriving repr]
type my_polyvar = [ int s | `T of string ] [@@deriving repr]
