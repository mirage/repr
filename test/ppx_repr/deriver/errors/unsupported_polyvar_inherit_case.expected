File "unsupported_polyvar_inherit_case.ml", line 2, characters 18-42:
Error: ppx_repr: inherited variant cases encountered in [ | int s
                                                        | `T of string ]. This is unsupported by ppx_repr.
